tests/validation/lib/aws.py

```diff
@@ -4,88 +4,36 @@
 import os
 import rsa
 import time
-
 from boto3.exceptions import Boto3Error
 from .cloud_provider import CloudProviderBase
 from .node import Node
 
 logging.getLogger('boto3').setLevel(logging.CRITICAL)
 logging.getLogger('botocore').setLevel(logging.CRITICAL)
 
-
 AWS_REGION = os.environ.get("AWS_REGION", "us-east-2")
 AWS_REGION_AZ = os.environ.get("AWS_REGION_AZ", "us-east-2a")
 AWS_SECURITY_GROUP = os.environ.get("AWS_SECURITY_GROUPS",
                                     'sg-0e753fd5550206e55')
 AWS_SUBNET = os.environ.get("AWS_SUBNET", "subnet-ee8cac86")
+AWS_HOSTED_ZONE_ID = os.environ.get("AWS_HOSTED_ZONE_ID", "")
 AWS_VPC_ID = os.environ.get("AWS_VPC_ID", "vpc-bfccf4d7")
-AWS_SECURITY_GROUPS = [AWS_SECURITY_GROUP]
 AWS_ACCESS_KEY_ID = os.environ.get("AWS_ACCESS_KEY_ID")
 AWS_SECRET_ACCESS_KEY = os.environ.get("AWS_SECRET_ACCESS_KEY")
 AWS_SSH_KEY_NAME = os.environ.get("AWS_SSH_KEY_NAME")
-AWS_CICD_INSTANCE_TAG = os.environ.get(
-    "AWS_CICD_INSTANCE_TAG", 'rancher-validation')
-AWS_INSTANCE_TYPE = os.environ.get("AWS_INSTANCE_TYPE", 't2.medium')
+AWS_CICD_INSTANCE_TAG = os.environ.get("AWS_CICD_INSTANCE_TAG",
+                                       'rancher-validation')
 AWS_IAM_PROFILE = os.environ.get("AWS_IAM_PROFILE", "")
-
-AWS_AMI = os.environ.get("AWS_AMI", "")
-AWS_USER = os.environ.get("AWS_USER", "ubuntu")
-AWS_HOSTED_ZONE_ID = os.environ.get("AWS_HOSTED_ZONE_ID", "")
+# by default the public Ubuntu 18.04 AMI is used
+AWS_DEFAULT_AMI = "ami-0d5d9d301c853a04a"
+AWS_DEFAULT_USER = "ubuntu"
+AWS_AMI = os.environ.get("AWS_AMI", AWS_DEFAULT_AMI)
+AWS_USER = os.environ.get("AWS_USER", AWS_DEFAULT_USER)
 AWS_VOLUME_SIZE = os.environ.get("AWS_VOLUME_SIZE", "50")
-AWS_WINDOWS_VOLUME_SIZE = os.environ.get("AWS_WINDOWS_VOLUME_SIZE", "100")
+AWS_INSTANCE_TYPE = os.environ.get("AWS_INSTANCE_TYPE", 't2.medium')
 
-PRIVATE_IMAGES = {
-    "rancheros-v1.5.1-docker-native": {
-        'image': 'ami-00769ca587d8e100c', 'ssh_user': 'rancher'},
-    "rancheros-v1.5.4-docker-native": {
-        'image': 'ami-022fc798437fb846a', 'ssh_user': 'rancher'},
-    "rancheros-v1.5.5-docker-native": {
-        'image': 'ami-002ab867b8b8591d5', 'ssh_user': 'rancher'},
-    "rhel-7.6-docker-native-113": {
-        'image': 'ami-04b9aacf7e1512c0b', 'ssh_user': 'ec2-user'},
-    "rhel-7.7-docker-native-113-selinux-off": {
-        'image': 'ami-03ff3301b6b68eaf8', 'ssh_user': 'ec2-user'},
-    "rhel-7.7-docker-native-113-selinux-on": {
-        'image': 'ami-069569219156fdf07', 'ssh_user': 'ec2-user'},
-    "rhel-7.7-docker-19.03-selinux-off": {
```

Comment:
Are we documenting these somewhere in the QA docs ? I just trying to figure how to know these values when defining `AWS_AMI` and `AWS_USER`

Reply:
when running `test_windows_provisioning`  `AWS_AMI` and `AWS_USER` are values for the Windows nodes. 
we don't want to introduce too many parameters in the Jenkins job so make the default values for Linux.